### DISCUSSION TITLE
Add `field_quote` for TextQueryBackend in Equal Expression

### DISCUSSION THREAD

**Discussion by nNipsx-Sec** [SigmaHQ/pySigma#49] [Ideas]
In some case of SIEM, the field have a whitespace example: `File Name`  so it's need add quote to define fields name. We can add this quote to some expression but can't add field quote to equal expression.

So my solution is add 1 `field_quote` like as a `str_quote`.

```
field_quote       : ClassVar[str] = "" 
```

**Comment by thomaspatzke** [comment]
Oh, yes, this is a gap...I will implement this in the TextQueryBackend base class!

**Reply by nNipsx-Sec** [reply]
Thanks it's need to Qradar backend, waiting for this.

**Comment by thomaspatzke** [comment]
Merged your PR, extended it and fixed some issues in other affected areas, now it's in release 0.6.2!

**Reply by nNipsx-Sec** [reply]
Great thanks for add more extended code.
